src/test-support/mocks/src/zone-ingresses/_/_overview.ts

```diff
@@ -80,15 +82,15 @@ export default ({ fake, env }: EndpointDependencies): MockResponder => (req) =>
                 },
                 version: {
                   kumaDp: {
-                    version: '1.2.1',
-                    gitTag: '1.2.1',
-                    gitCommit: 'e88ec407e669c47d3dc9ef32fcde60e2f31c0c4d',
-                    buildDate: '2021-06-30T14:32:58Z',
-                  },
-                  envoy: {
-                    version: '1.18.3',
-                    build: '98c1c9e9a40804b93b074badad1cdf284b47d58b/1.18.3/Clean/RELEASE/BoringSSL',
+                    version: fake.system.semver(),
+                    gitTag: fake.system.semver(),
+                    gitCommit: fake.git.commitSha(),
+                    buildDate: fake.kuma.date(),
                   },
+                  envoy: ((sha, version) => ({
+                    version,
+                    build: `${sha}/${version}/Clean/RELEASE/BoringSSL`,
```

Comment:
I tried using `fake.system.directoryPath` here but it didn't work for some reason, don't think its important for this to be random though 🤷